nucypher/cli/actions.py

```diff
@@ -108,6 +108,40 @@ def unlock_nucypher_keyring(emitter, password: str, character_configuration: Cha
         raise character_configuration.keyring.AuthenticationFailed
 
 
+def load_static_nodes(domain: str, filepath: str = None) -> List[str]:
+    """
+    Non-invasively read teacher-uris from a JSON configuration file keyed by domain name.
+    and return them as a list of URI strings.
+    """
+
+    # Get filepath
+    if not filepath:
+        filepath = os.path.join(DEFAULT_CONFIG_ROOT, 'static-nodes.json')
+
+    # Read
+    try:
+        with open(filepath, 'r') as file:
+            data = file.read()
+    except FileNotFoundError:
+        # If there is no static nodes file, there are no results
+        return list()
+
+    # Decode
+    try:
+        static_nodes = json.loads(data)
+    except JSONDecodeError:
+        raise RuntimeError(f"Static nodes configuration at '{filepath}' contains invalid JSON.")
```

Comment:
If you want, you could save some lines by doing:
```python
try:
    with open(filepath, 'r') as file:
        static_nodes = json.load(file)
except FileNotFoundError:
    return list()
except JSONDecodeError:
    raise RuntimeError(f"Static nodes configuration at '{filepath}' contains invalid JSON.")
```

Reply:
:heavy_check_mark: :+1: